client/src/index.jsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+
+class Menu extends React.Component {
+	constructor() {
+		super();
+	}
+
+	render() {
```

Comment:
spacing is funky in this file (too many).  airbnb style recommends 2 spaces for each indent.  eslint would give you suggestions how to modify them